site/docs/spark.md

```diff
@@ -15,9 +15,9 @@
  - limitations under the License.
  -->
 
-# Spark
+# Apache Spark
 
-Iceberg uses Spark's DataSourceV2 API for data source and catalog implementations. Spark DSv2 is an evolving API with different levels of support in Spark versions.
+Apache Iceberg uses Spark's DataSourceV2 API for data source and catalog implementations. Spark DSv2 is an evolving API with different levels of support in Spark versions.
```

Comment:
I think this is another example of where we don't want to use "Apache" in headings. I think instead the heading should be "Spark" and the text should be "Iceberg uses Apache Spark's DataSourceV2 API . . .". Iceberg also doesn't need to be "Apache Iceberg" since the first reference is already "Apache Iceberg" in the site header.

Reply:
Valid point, updated it.